apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo/gradle/internal/DefaultCompilerParams.kt

```diff
@@ -42,11 +43,16 @@ abstract class DefaultCompilerParams @Inject constructor(objects: ObjectFactory)
   abstract override val alwaysGenerateTypesMatching: SetProperty<String>
 
   init {
-    // see https://github.com/gradle/gradle/issues/7485
-    // TODO replace with `convention(null)` when we can target Gradle 6.2
-    customTypeMapping.set(null as Map<String, String>?)
-    sealedClassesForEnumsMatching.set(null as List<String>?)
-    alwaysGenerateTypesMatching.set(null as Set<String>?)
+    if (GradleVersion.current() >= GradleVersion.version("6.2")) {
```

Comment:
Seeing as Gradle is not up to 6.7 with 7 around the corner it doesn't feel unreasonable to require 6.2+

Reply:
Yup. At the same time, [there are still users on Gradle 5.6](https://github.com/apollographql/apollo-android/issues/2536) so if we can make this work for them, I think it's not too much of a price to pay for now.